webapp/crashstats/libbugzilla.py

```diff
@@ -69,6 +68,33 @@ def frame_generator(thread):
     return frames
 
 
+def java_exception_to_frames(stack):
```

Comment:
This is new. I modeled it after the `minidump_thread_to_frames`, but I think we're guaranteed to have actual values for the different fields. Instead of verifying that by looking at the crash reporter code for Android, I figured we could be defensive and handle the cases where data may not exist.